src/backend/distributed/commands/function.c

```diff
@@ -662,10 +693,21 @@ UpdateFunctionDistributionInfo(const ObjectAddress *distAddress,
 			colocationIdList = list_make1_int(*colocationId);
 		}
 
+		if (forcePushdownAddress == NULL)
+		{
+			bool forcePushdown = false;
+			forcePushdownList = list_make1((bool *) &forcePushdown);
```

Comment:
this is weird, the bool goes out of scope one line below.

Reply:
hmm, you must be looking at some old code?

Reply:
Nope, you are right, I did this in two places, I fixed in one place. Anyway, I revamping this part.